transpiler/src/main/java/org/jsweet/transpiler/extension/Java2TypeScriptAdapter.java

```diff
@@ -1289,6 +1288,41 @@ public boolean substituteMethodInvocation(MethodInvocationElement invocationElem
                     printTarget(invocationElement.getTargetExpression());
                     print(")");
                     return true;
+                case "getFields":
+                case "getDeclaredFields":
+                    printMacroName(targetMethodName);
+                    if (targetType instanceof DeclaredType) {
+                        List<? extends TypeMirror> typeArgs = ((DeclaredType) targetType).getTypeArguments();
+                        if (typeArgs != null && typeArgs.size() > 0 && !util().isType(typeArgs.get(0), Object.class)) {
+                            TypeMirror targetClassType = typeArgs.get(0);
+                            Class<?> targetClass = util().getTypeClass(targetClassType);
+
+                            if (targetClass != null) {
+                                Field[] fields;
+                                if (targetMethodName.equals("getDeclaredFields")) {
+                                    fields = targetClass.getDeclaredFields();
+                                } else {
+                                    fields = targetClass.getFields();
+                                }
+                                System.out.println("tesseeee " + targetMethodName + " == " + fields.length);
```

Comment:
remove

Reply:
That's a complicated one. Did you check it was compatible with J4TS? What if we want to support it in J4TS?